database/mysql/mysql.go

```diff
@@ -162,9 +172,9 @@ func (m *Mysql) Lock() error {
 		return err
 	}
 
-	query := "SELECT GET_LOCK(?, 1)"
+	query := "SELECT GET_LOCK(?, 10)"
```

Comment:
We should probably make the timeout configurable, but I think it's fine to make that change in a separate PR